### PR TITLE
Translating en/model_doc folder docs to Japanese(from `blip` to `clap`) 🇯🇵

### LINKED ISSUE #27669: [i18n-jp] Translating `en/model_doc` folder docs to Japanese 🇯🇵
Hi!

Let's bring the documentation to all the japanese-speaking community 🌐 (currently 0 out of 267 complete)

Who would want to translate? Please follow the 🤗 [TRANSLATING guide](https://github.com/huggingface/transformers/blob/main/docs/TRANSLATING.md). Here is a list of the files ready for translation. Let us know in this issue if you'd like to translate any, and we'll add your name to the list.

Some notes:

* Please translate using an informal tone (imagine you are talking with a friend about transformers 🤗).
* Please translate in a gender-neutral way.
* Add your translations to the folder called `ja` inside the [source folder](https://github.com/huggingface/transformers/tree/main/docs/source).
* Register your translation in `<ja/_toctree.yml`; please follow the order of the [English version](https://github.com/huggingface/transformers/blob/main/docs/source/en/_toctree.yml).
* Once you're finished, open a pull request and tag this issue by including #issue-number in the description, where issue-number is the number of this issue. Please ping @stevhliu and @MKhalusova for review.
* 🙋 If you'd like others to help you with the translation, you can also post in the 🤗 [forums](https://discuss.huggingface.co/).

## Model_doc section

- [ ] [blip-2.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/blip-2.md) #27673 
- [ ] [bloom.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/bloom.md) #27673 
- [ ] [bort.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/bort.md). #27673 
- [ ] [bridgetower.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/bridgetower.md). #27673 
- [ ] [bros.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/bros.md). #27673 
- [ ] [byt5.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/byt5.md). #27673 
- [ ] [camembert.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/camembert.md). #27673 
- [ ] [canine.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/canine.md). #27673 
- [ ] [chinese_clip.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/chinese_clip.md). #27673 
- [ ] [clap.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/clap.md). #27673

### PR DESCRIPTION
**Pull request by rajveer43**
# What does this PR do?

Fixes #27669

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Documentation: @stevhliu and @MKhalusova

### CODE REVIEW

File: docs/source/ja/model_doc/blip-2.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+## Usage tips
+
+- BLIP-2 は、画像とオプションのテキスト プロンプトを指定して条件付きテキストを生成するために使用できます。推論時には、 [`generate`] メソッドを使用することをお勧めします。
+- [`Blip2Processor`] を使用してモデル用の画像を準備し、予測されたトークン ID をデコードしてテキストに戻すことができます。
+
+## Resources
+
+BLIP-2 の使用を開始するのに役立つ公式 Hugging Face およびコミュニティ (🌎 で示されている) リソースのリスト。
+
+- 画像キャプション、ビジュアル質問応答 (VQA)、およびチャットのような会話のための BLIP-2 のデモ ノートブックは、[こちら](https://github.com/NielsRogge/Transformers-Tutorials/tree/master/BLIP-2) にあります。 ）。
```
Comment (stevhliu) [inline docs/source/ja/model_doc/blip-2.md:47]:
```suggestion
- 画像キャプション、ビジュアル質問応答 (VQA)、およびチャットのような会話のための BLIP-2 のデモ ノートブックは、[こちら](https://github.com/NielsRogge/Transformers-Tutorials/tree/master/BLIP-2) にあります。
```

File: docs/source/ja/model_doc/bort.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+このモデルはメンテナンス モードのみであり、コードを変更する新しい PR は受け付けられません。
+
+このモデルの実行中に問題が発生した場合は、このモデルをサポートしていた最後のバージョン (v4.30.0) を再インストールしてください。
+これを行うには、コマンド `pip install -U Transformers==4.30.0` を実行します。
+
+</Tip>
+
+## Overview
+
+BORT モデルは、[Optimal Subarchitecture Extraction for BERT](https://arxiv.org/abs/2010.10499) で提案されました。
```
Comment (stevhliu) [inline docs/source/ja/model_doc/bort.md:30]: Author names also got lost in translation here

File: docs/source/ja/model_doc/bros.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+`BrosForTokenClassification`と`BrosSpadeEEForTokenClassification`は基本的に同じジョブを実行します。ただし、`BrosForTokenClassification`は入力トークンが完全にシリアル化されていることを前提としています (トークンは 2D 空間に存在するため、これは非常に困難な作業です)。一方、`BrosSpadeEEForTokenClassification`は 1 つのトークンから次の接続トークンを予測するため、シリアル化エラーの処理をより柔軟に行うことができます。
+
+`BrosSpadeELForTokenClassification` はエンティティ内のリンク タスクを実行します。これら 2 つのエンティティが何らかの関係を共有する場合、(あるエンティティの) 1 つのトークンから (別のエンティティの) 別のトークンへの関係を予測します。
+
+BROS は、明示的な視覚機能に依存せずに、FUNSD、SROIE、CORD、SciTSR などの Key Information Extraction (KIE) ベンチマークで同等以上の結果を達成します。
+
+論文の要約は次のとおりです。
+
+*文書画像からの重要情報抽出 (KIE) には、2 次元 (2D) 空間におけるテキストの文脈的および空間的意味論を理解する必要があります。最近の研究の多くは、文書画像の視覚的特徴とテキストおよびそのレイアウトを組み合わせることに重点を置いた事前トレーニング済み言語モデルを開発することで、この課題を解決しようとしています。一方、このペーパーでは、テキストとレイアウトの効果的な組み合わせという基本に立ち返ってこの問題に取り組みます。具体的には、BROS (BERT Relying On Spatality) という名前の事前トレーニング済み言語モデルを提案します。この言語モデルは、2D 空間内のテキストの相対位置をエンコードし、エリア マスキング戦略を使用してラベルのないドキュメントから学習します。 2D 空間内のテキストを理解するためのこの最適化されたトレーニング スキームにより、BROS は、視覚的な特徴に依存することなく、4 つの KIE ベンチマーク (FUNSD、SROIE*、CORD、および SciTSR) で以前の方法と比較して同等以上のパフォーマンスを示しました。また、この論文では、KIE タスクにおける 2 つの現実世界の課題 ((1) 間違ったテキスト順序によるエラーの最小化、および (2) 少数の下流例からの効率的な学習) を明らかにし、以前の方法に対する BROS の優位性を実証します。
```
Comment (stevhliu) [inline docs/source/ja/model_doc/bros.md:36]:
```suggestion
*文書画像からの重要情報抽出 (KIE) には、2 次元 (2D) 空間におけるテキストの文脈的および空間的意味論を理解する必要があります。最近の研究の多くは、文書画像の視覚的特徴とテキストおよびそのレイアウトを組み合わせることに重点を置いた事前トレーニング済み言語モデルを開発することで、この課題を解決しようとしています。一方、このペーパーでは、テキストとレイアウトの効果的な組み合わせという基本に立ち返ってこの問題に取り組みます。具体的には、BROS (BERT Relying On Spatality) という名前の事前トレーニング済み言語モデルを提案します。この言語モデルは、2D 空間内のテキストの相対位置をエンコードし、エリア マスキング戦略を使用してラベルのないドキュメントから学習します。 2D 空間内のテキストを理解するためのこの最適化されたトレーニング スキームにより、BROS は、視覚的な特徴に依存することなく、4 つの KIE ベンチマーク (FUNSD、SROIE*、CORD、および SciTSR) で以前の方法と比較して同等以上のパフォーマンスを示しました。また、この論文では、KIE タスクにおける 2 つの現実世界の課題 ((1) 間違ったテキスト順序によるエラーの最小化、および (2) 少数の下流例からの効率的な学習) を明らかにし、以前の方法に対する BROS の優位性を実証します。*
```

File: docs/source/ja/model_doc/camembert.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+
+-->
+
+# CamemBERT
+
+## Overview
+
+CamemBERT モデルは、[CamemBERT: a Tasty French Language Model](https://arxiv.org/abs/1911.03894) で提案されました。
```
Comment (stevhliu) [inline docs/source/ja/model_doc/camembert.md:21]: Authors missing

File: docs/source/ja/model_doc/camembert.md [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+論文の要約は次のとおりです。
+
+*事前トレーニングされた言語モデルは現在、自然言語処理で広く普及しています。成功にもかかわらず、利用可能なほとんどの
+モデルは英語のデータ、または複数言語のデータの連結でトレーニングされています。これにより、
+このようなモデルの実際の使用は、英語を除くすべての言語で非常に限られています。フランス人にとってこの問題に対処することを目指して、
+Bi-direction Encoders for Transformers (BERT) のフランス語版である CamemBERT をリリースします。測定します
+複数の下流タスク、つまり品詞タグ付けにおける多言語モデルと比較した CamemBERT のパフォーマンス
+依存関係解析、固有表現認識、自然言語推論。 CamemBERT は最先端技術を向上させます
+検討されているほとんどのタスクに対応します。私たちは、研究と
+フランス語 NLP の下流アプリケーション*。
```
Comment (stevhliu) [inline docs/source/ja/model_doc/camembert.md:35]:
```suggestion
フランス語 NLP の下流アプリケーション。*
```

File: docs/source/ja/model_doc/camembert.md [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+入力と出力に関する情報として。
+
+</Tip>
+
+## Resources
+
+- [テキスト分類タスクガイド](../tasks/sequence_classification)
+- [トークン分類タスクガイド](../tasks/token_classification)
+- [質問回答タスク ガイド](../tasks/question_answering)
+- [因果言語モデリング タスク ガイド](../tasks/ language_modeling)
```
Comment (stevhliu) [inline docs/source/ja/model_doc/camembert.md:52]:
```suggestion
- [因果言語モデリング タスク ガイド](../tasks/language_modeling)
```

File: docs/source/ja/model_doc/canine.md [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+
+-->
+
+# CANINE
+
+## Overview
+
+CANINE モデルは、[CANINE: 言語用の効率的なトークン化フリーのエンコーダーの事前トレーニング](https://arxiv.org/abs/2103.06874)、Jonathan H. Clark、Dan Garrette、Iulia Turc、John Wieting 著。その
+明示的なトークン化ステップ (バイト ペアなど) を使用せずに Transformer をトレーニングする最初の論文の 1 つ
+エンコーディング (BPE)、WordPiece または SentencePiece)。代わりに、モデルは Unicode 文字レベルで直接トレーニングされます。
```
Comment (stevhliu) [inline docs/source/ja/model_doc/canine.md:23]: Missing the opening `(` here for BPE, WordPiece and SentencePiece

File: docs/source/ja/model_doc/chinese_clip.md [hunk 8]
```diff
@@ -0,0 +1,112 @@
...
+⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+
+-->
+
+# Chinese-CLIP
+
+## Overview
+
+Chinese-CLIP モデルは、An Yang、Junshu Pan、Junyang Lin、Rui Men、Yichang Zhang、Jingren によって [中国語 CLIP: Contrastive Vision-Language Pretraining in Chinese](https://arxiv.org/abs/2211.01335) で提案されました。周、張周。
```
Comment (stevhliu) [inline docs/source/ja/model_doc/chinese_clip.md:21]: Missing an author here

File: docs/source/ja/model_doc/chinese_clip.md [hunk 9]
```diff
@@ -0,0 +1,112 @@
...
+Chinese-CLIP は、中国語の画像とテキストのペアの大規模なデータセットに対する CLIP (Radford et al., 2021) の実装です。クロスモーダル検索を実行できるほか、ゼロショット画像分類、オープンドメインオブジェクト検出などのビジョンタスクのビジョンバックボーンとしても機能します。オリジナルの中国語-CLIPコードは[このリンクで](https: //github.com/OFA-Sys/ Chinese-CLIP)。
```
Comment (stevhliu) [inline docs/source/ja/model_doc/chinese_clip.md:22]:
```suggestion
Chinese-CLIP は、中国語の画像とテキストのペアの大規模なデータセットに対する CLIP (Radford et al., 2021) の実装です。クロスモーダル検索を実行できるほか、ゼロショット画像分類、オープンドメインオブジェクト検出などのビジョンタスクのビジョンバックボーンとしても機能します。オリジナルの中国語-CLIPコードは[このリンクで](https://github.com/OFA-Sys/Chinese-CLIP)。
```

File: docs/source/ja/model_doc/byt5.md [hunk 10]
```diff
@@ -0,0 +1,154 @@
...
+⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+
+-->
+
+# ByT5
+
+## Overview
+
+ByT5 モデルは、[ByT5: 事前トレーニングされたバイト間モデルによるトークンフリーの未来へ](https://arxiv.org/abs/2105.13626) で、Linting Xue、Aditya Barua、Noah Constant、Rami によって提示されました。アル・ルフォウ、シャラン・ナラン、ミヒル
```
Comment (stevhliu) [inline docs/source/ja/model_doc/byt5.md:21]: I noticed in some of the translations, you don't translate the paper title and in others it is translated. I think for consistency, let's keep the paper titles untranslated.

**Review by stevhliu — COMMENTED** [review]
Thanks for starting translations on the next batch of models!

File: docs/source/ja/model_doc/blip-2.md [hunk 11]
```diff
@@ -0,0 +1,90 @@
...
+⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+
+-->
+
+# BLIP-2
+
+## Overview
+
+BLIP-2 モデルは、[BLIP-2: Bootstrapping Language-Image Pre-training with Frozen Image Encoders and Large Language Models](https://arxiv.org/abs/2301.12597) で提案されました。
```
[the remark this answers is not in the document]
Comment (rajveer43) [reply docs/source/ja/model_doc/blip-2.md:21]: okay doing that

File: docs/source/ja/model_doc/bort.md (hunk 2 above)
Comment (rajveer43) [reply docs/source/ja/model_doc/bort.md:30]: the names got translated too

File: docs/source/ja/model_doc/byt5.md (hunk 10 above)
Comment (rajveer43) [reply docs/source/ja/model_doc/byt5.md:21]: got it

File: docs/source/ja/model_doc/chinese_clip.md [hunk 12]
```diff
@@ -0,0 +1,112 @@
...
+⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+
+-->
+
+# Chinese-CLIP
+
+## Overview
+
+Chinese-CLIP An Yang, Junshu Pan, Junyang Lin, Rui Men, Yichang Zhang, Jingren Zhou, Chang Zhou [中国語 CLIP: Contrastive Vision-Language Pretraining in Chinese](https://arxiv.org/abs/2211.01335) で提案されました。周、張周。
```
Comment (stevhliu) [inline docs/source/ja/model_doc/chinese_clip.md:21]:
```suggestion
Chinese-CLIP An Yang, Junshu Pan, Junyang Lin, Rui Men, Yichang Zhang, Jingren Zhou, Chang Zhou [Chinese CLIP: Contrastive Vision-Language Pretraining in Chinese](https://arxiv.org/abs/2211.01335) で提案されました。周、張周。
```

File: docs/source/ja/model_doc/canine.md [hunk 13]
```diff
@@ -0,0 +1,143 @@
...
+⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+
+-->
+
+# CANINE
+
+## Overview
+
+CANINE モデルは、[CANINE: 言語用の効率的なトークン化フリーのエンコーダーの事前トレーニング](https://arxiv.org/abs/2103.06874)、Jonathan H. Clark、Dan Garrette、Iulia Turc、John Wieting 著。その
```
Comment (stevhliu) [inline docs/source/ja/model_doc/canine.md:21]:
```suggestion
CANINE モデルは、[CANINE: Pre-training an Efficient Tokenization-Free Encoder for Language
Representation](https://arxiv.org/abs/2103.06874)、Jonathan H. Clark、Dan Garrette、Iulia Turc、John Wieting 著。その
```

File: docs/source/ja/model_doc/bros.md [hunk 14]
```diff
@@ -0,0 +1,113 @@
...
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# BROS
+
+## Overview
+
+BROS モデルは、Teakgyu Hon、Donghyun Kim、Mingi Ji によって [BROS: A Pre-trained Language Model Focusing on Text and Layout for Better Key Information Extraction from Documents](https://arxiv.org/abs/2108.04539) で提案されました。 、ファン・ウォンソク、ナム・デヒョン、パク・ソンレ。
```
Comment (stevhliu) [inline docs/source/ja/model_doc/bros.md:17]: These authors are still missing: Wonseok Hwang, Daehyun Nam, Sungrae Park

File: docs/source/ja/model_doc/bridgetower.md [hunk 15]
```diff
@@ -0,0 +1,171 @@
...
+⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+
+-->
+
+# BridgeTower
+
+## Overview
+
+BridgeTower モデルは、Xiao Xu、Chenfei Wu、Shachar Rosenman、Vasudev Lal、Wanxiang Che、Nan によって [BridgeTower: Building Bridges Between Encoders in Vision-Language Representative Learning](https://arxiv.org/abs/2206.08657) で提案されました。ドゥアン。このモデルの目標は、
```
Comment (stevhliu) [inline docs/source/ja/model_doc/bridgetower.md:21]:
```suggestion
BridgeTower モデルは、Xiao Xu、Chenfei Wu、Shachar Rosenman、Vasudev Lal、Wanxiang Che、Nan Duan [BridgeTower: Building Bridges Between Encoders in Vision-Language Representative Learning](https://arxiv.org/abs/2206.08657) で提案されました。ドゥアン。このモデルの目標は、
```

**Review by stevhliu — COMMENTED** [review]
A few more accidentally translated things, but otherwise should be good to go after they're fixed

**Review by stevhliu — APPROVED** [review]
LGTM! Can you try running `make fixup` and `pip uninstall black && pip install -U ruff==0.1.5` to fix the failing CI test?

**Comment by rajveer43** [comment]
image
this is the error I received should edit that file? @stevhliu

**Pull request merged**